### DISCUSSION TITLE
Maximum length and minimum length for fields

### DISCUSSION THREAD

**Discussion by nithin-techforce** [directus/directus#5552] [Feature Requests]
Hi , i have a field "contact" in my directus collection where i want to set its minimum length should be 10 as indians contact numbers are ten digits ,but i see no option for that i see minimun value and maximum value which  are not relavant i think
and another issues is i am unable to export more that 100 items of a collection to an excel sheet

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
